core/src/main/java/org/mobilitydata/gtfsvalidator/notice/NonAsciiOrNonPrintableCharNotice.java

```diff
@@ -18,14 +18,15 @@
 
 import com.google.common.collect.ImmutableMap;
 
-/** A row in the input file has a different number of values than specified by the CSV header. */
+/** ID value contains something different from printable ASCII characters. */
 public class NonAsciiOrNonPrintableCharNotice extends ValidationNotice {
   public NonAsciiOrNonPrintableCharNotice(String filename, long csvRowNumber, String columnName) {
     super(
         ImmutableMap.of(
             "filename", filename,
             "csvRowNumber", csvRowNumber,
-            "columnName", columnName));
+            "columnName", columnName),
+        SeverityLevel.INFO);
```

Comment:
Shouldn't this be `SeverityLevel.WARNING`?

Reply:
OK, done.